bindings/matrix-sdk-ffi/Cargo.toml

```diff
@@ -57,7 +57,7 @@ language-tags = "0.3.2"
 log-panics = { version = "2", features = ["with-backtrace"] }
 paranoid-android = "0.2.1"
 
-[target.'cfg(target_os = "android")'.dependencies.matrix-sdk]
+[target.'cfg(any(target_os = "android", target_os = "ios", target_os = "macos"))'.dependencies.matrix-sdk]
```

Comment:
Since this crate is in practice only used for these two target platforms right now, it might be better to just make this a regular `[dependencies]` entry instead of a target-specific one, and delete the block w/ `native-tls` in it.

Reply:
Thanks for the hint, I was not aware of that. Will proceed like that